annotation-processor/pom.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.gaul</groupId>
+    <artifactId>modernizer-maven-parent</artifactId>
+    <version>1.0-hubspot-SNAPSHOT</version>
+  </parent>
+
+  <groupId>org.gaul</groupId>
+  <artifactId>annotation-processor</artifactId>
+  <version>1.0-hubspot-SNAPSHOT</version>
+  <properties>
+    <auto-service.version>1.0-rc4</auto-service.version>
```

Comment:
We shouldn't need a variable because this is used in one place so updating it is easy